sensors/sensors.smithy

```diff
@@ -0,0 +1,121 @@
+// Definition of the sensor measurement interface `wasmcloud:sensors`
+
+// Provide helpful metadata used by code generators
+metadata package = [{
```

Comment:
Does this file need to exist when docs/idl/org.wasmcloud/sensors.smithy exists?

Reply:
Yes, the `docs/idl/org.wasmcloud/sensors.smithy` is used by the github action to generate the docs....

This repo doesnt really have sane automation....the GHAs create and push the rust crates, but the go modules are added by PR.  

So TLDR, yea, the `sensors/sensors.smithy` needs to be there 

To be clear, this PR doesnt create the Rust crate